docs-src/tutorials/04-cookbook.md

```diff
@@ -54,8 +54,15 @@ being show.
 ```javascript
 when: {
   show() {
-    const currentStepElement = shepherd.currentStep.el;
-    const header = currentStepElement.querySelector('.shepherd-header');
+    const currentStep = Shepherd.activeTour?.getCurrentStep()
+    if (!currentStep)
+      return
+    const currentStepElement = currentStep.getElement()
+    if (!currentStepElement)
+      return
+    const header = currentStepElement.querySelector('.shepherd-header')
+    if (!header)
+      return
```

Comment:
```suggestion
    const currentStep = Shepherd.activeTour?.getCurrentStep();
    const currentStepElement = currentStep?.getElement();
    const header = currentStepElement?.querySelector('.shepherd-header');
```

I think optional chaining looks a lot cleaner here than returning if each thing doesn't exist. What do you think?